meta-mender-tegra/conf/layer.conf

```diff
@@ -0,0 +1,12 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+		${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "meta-mender-tegra-jetpack5"
```

Comment:
Shouldn't this be `meta-mender-tegra` now?
Same for the overrides that follow.

Reply:
Good catch. I have posted a new patch that fixes this.